cli/internal/runsummary/task_summary.go

```diff
@@ -41,56 +41,21 @@ type TaskEnvVarSummary struct {
 	Global     []string `json:"global"`
 }
 
-// toSinglePackageTask converts a TaskSummary into a singlePackageTaskSummary
-func (ht *TaskSummary) toSinglePackageTask() singlePackageTaskSummary {
-	dependencies := make([]string, len(ht.Dependencies))
-	for i, depencency := range ht.Dependencies {
+// cleanForSinglePackage converts a TaskSummary to remove references to workspaces
+func (ts *TaskSummary) cleanForSinglePackage() {
+	dependencies := make([]string, len(ts.Dependencies))
+	for i, depencency := range ts.Dependencies {
```

Comment:
```suggestion
	for i, dependency := range ts.Dependencies {
```

Reply:
ty, updated in another commit since i had to update the usage also :D